examples/Example.h

```diff
@@ -368,7 +368,7 @@ struct WgWindow : Window
     {
         if (!initialized) return;
 
-        window = SDL_CreateWindow("ThorVG Example (WebGPU)", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, width, height, SDL_WINDOW_HIDDEN);
+        window = SDL_CreateWindow("ThorVG Example (WebGPU)", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, width, height, SDL_WINDOW_SHOWN);
```

Comment:
@SergeyLebedkin `HIDDEN` should work correctly as well. Please double-check if it functions properly without the change.

Reply:
> @SergeyLebedkin `HIDDEN` should work correctly as well. Please double-check if it functions properly without the change.

OK.. will check on synthetic tests